graphql/schema.graphql

```diff
@@ -402,6 +402,7 @@ type Patch {
   time: PatchTime
   taskCount: Int
   baseVersionID: String
+  parameters: [Parameter!]!
```

Comment:
Will every patch always have parameters, or can it be null? I would test this out with a page that doesn't have any parameters to see what it returns in that case.

Reply:
Ah interesting -- the way the fromService code is written the list is always initialized even if there are no parameters, but logically it can be null so I'm going to remove one of the ! in case the FromService code ever changes.